Author: Thomas Krennwallner <tkren@kr.tuwien.ac.at>
Modified for 3.0.4 by Davide Iosca
Description: Fix invalid use of incomplete type Domain compile error.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=652787
diff -aur gringo-3.0.4-source.old/libgringo/src/parser.cpp gringo-3.0.4-source/libgringo/src/parser.cpp
--- gringo-3.0.4-source.old/libgringo/src/parser.cpp	2012-01-26 13:08:59.000000000 +0100
+++ gringo-3.0.4-source/libgringo/src/parser.cpp	2012-02-05 17:52:50.405166419 +0100
@@ -24,6 +24,7 @@
 #include <gringo/constterm.h>
 #include <gringo/prgvisitor.h>
 #include <gringo/predlit.h>
+#include <gringo/domain.h>
 #include <gringo/rule.h>
 #include <gringo/streams.h>
 #include <gringo/grounder.h>
